tools/conformance/djxl_conformance.cc

```diff
@@ -61,21 +61,40 @@ bool SaveFile(const char* filename, std::vector<uint8_t> data) {
 struct ImageArray {
   uint32_t xsize, ysize;
   uint32_t num_channels;
+  uint32_t num_color_channels;
+  uint32_t num_extra_channels;
 
   // An array of "frames", where each frame is a 3D array of samples with
   // dimensions (ysize, xsize, channel).
   std::vector<std::vector<uint8_t>> frames;
+  // Extra channels, as an array of frames, where each frame is an array of 2D
+  // arrays of samples
```

Comment:
what are the dimensions of these 2D arrays? explain how is this interleaved in the .npy

Reply:
Updated the documentation